apiserver/resources/server_addpending_test.go

```diff
@@ -293,24 +292,24 @@ func (s *AddPendingResourcesSuite) TestWithURLUpload(c *gc.C) {
 //func (s *AddPendingResourcesSuite) TestUnknownResource(c *gc.C) {
```

Comment:
Not sure what the background here is but it looks like one of the other (enabled) tests already covers this.